vmngclient/endpoints/configuration_feature_profile.py

```diff
@@ -46,6 +48,14 @@ class FeatureProfileCreationResponse(BaseModel):
     id: str
 
 
+class ParcelCreationResponse(BaseModel):
```

Comment:
could be just ParcelId class for both put, post responses:
```python
class ParcelId(BaseModel):
    id: str = Field(alias="parcelId")
```
Modification should be easy even if return type diverges at some point.